src/libraries/System.Private.Xml/src/System/Xml/Schema/XsdDateTime.cs

```diff
@@ -688,7 +690,7 @@ private void IntToCharArray(char[] text, int start, int value, int digits)
         }
```

Comment:
Do we even need this helper anymore?  Can we instead use int.TryFormat?

Reply:
removed and using AppendSpanFormattable